src/data/list/basic.lean

```diff
@@ -4250,6 +4250,12 @@ nodup_pmap (λ _ _ _ _, fin.veq_of_eq) (nodup_range _)
 @[simp] lemma length_fin_range (n : ℕ) : (fin_range n).length = n :=
 by rw [fin_range, length_pmap, length_range]
 
+@[simp, to_additive list.sum_range_succ]
+theorem prod_range_succ {α : Type u} [monoid α] (f : ℕ → α) (n : ℕ) :
+((range (succ n)).map f).prod = ((range n).map f).prod * f n :=
```

Comment:
Indent this line with two spaces and replace `(succ n)` with `n.succ` (that will remove one pair of brackets)

Reply:
Done!